_data/sesalomon/sesalomon.md

```diff
@@ -0,0 +1,7 @@
+---
+name: S.E.S. 
+institution: Case Western Reserve University 
+profile_pic: cwrulogo.png 
+quote: "We are an impossibility in an impossible universe." Ray Bradbury 
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format